frontend-service.tf

```diff
@@ -0,0 +1,33 @@
+resource "github_repository" "frontend-service" {
+  name        = "frontend-service"
```

Comment:
Can we put this into the dataworks namespace please: `dataworks-frontend-service` or something even more descriptive (but I don't know what that would be...naming is really hard!). The reason for this is that github.com/dwp is used by much more than just us, so `frontend-service` is too generic to understand what the repo contains/who it's useful for.

Maybe not `dataworks` but perhaps `guacomole-frontend-service`? I dunno.